class.jetpack-modules-list-table.php

```diff
@@ -76,7 +77,18 @@ function js_templates() {
 			if ( data.items.length ) {
 			_.each( data.items, function( item, key, list ) {
 				if ( item === undefined ) return;
-				#>
+				if ( jetpackModulesData.coreIconAvailable && 'site-icon' == item.module ) { #>
+				<tr class="jetpack-module <# if ( ++i % 2 ) { #> alternate<# } #>" id="site-icon-deprecated">
```

Comment:
The only real purpose of the ID in this case is a target for the genericon, so making this reusable wouldn't really work for other modules.